scripts/update_theme_docs.py

```diff
@@ -97,13 +121,16 @@ def contrast_markdown_table(color_cls: Type, background_color: str) -> list[list
 
             # Calculate contrast
             contrast = contrast_ratio(hex_to_rgb(value), hex_to_rgb(background_color))
-            rrggbb = hexstr_without_hash(value)
+            rrggbb = hexstr_without_hash(value).lower()
+
+            make_square_png("#" + rrggbb)
```

Comment:
Would be nice to keep contrast_markdown_table as a pure function but this file is also just a script so, eh :)

Reply:
Nah, you are right, I can refactor.